planetiler-core/src/main/java/com/onthegomap/planetiler/mbtiles/Mbtiles.java

```diff
@@ -93,6 +94,12 @@ public final class Mbtiles implements TileArchive {
   private PreparedStatement getTileStatement = null;
   private final boolean compactDb;
 
+  /** Inserts will be ordered to match the MBTiles index (TMS) */
```

Comment:
In general I don't put javadoc on inhereted methods since they inherit javadoc from the parent.